2.adoc

```diff
@@ -72,5 +72,229 @@ Optional description
 . Minus A
 . Minus B
 
+=== GitBook
+
+Desktopový (free) a online (placený) editor pro markupové dokumenty ve formátu GFM (GitHub Flavored Markdown) a AsciiDoc.
+
+Navíc k editaci dokumentů umožňuje správu celých knih. Resp. používá metaforu knihy na rozdělení projektu do kapitol, případně do souborů.
+
+Na pozadí GitBook verzuje dokumenty pomocí gitu. Prohlížení změn dokumentu je tak pohled nad git commity.
+
+link:+https://www.gitbook.com/editor+[GitBook]
+
+==
+
+Klikání do preview neodscrolluje markup zdroj do správné pozice, což může být nepříjemné obzvláště u delších dokumentů.
```

Comment:
Tohle je znovu uvedene (a vice popsane) v sekci Scrollovani, tak bych to tady asi vyhodil..

Reply:
fixed;  to bylo celé nějaké mimo kontext